spec/v2/index.bs

```diff
@@ -1993,7 +1994,7 @@ The <dfn>PF Update Event</dfn> is defined as a JSON-encoded CloudEvent event wit
 <pre highlight=json>
 {
   "type": "org.wbcsd.pathfinder.ProductFootprint.Published.v1",
-  "specversion": "1.0",
+  "specversion": "[VERSION]",
```

Comment:
this must stay as is as it is referencing cloudevents

Reply:
Thank you! I was a bit unsure about these.